weaveworks/eksctl
# Add flag to enable Simple System Manager (SSM) for easy SSH access

### Description
This PR add the optional to enable [Simple Systems Manager (SSM)](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html) by passing the `--enable-ssm` flag. Once SSM is installed you can easily ssh onto nodes in the cluster by running `aws ssm start-session --target <node-id> --region <region>`

#### Manually Tested
- [X] AL2
  - Ubuntu images already have SSM pre-installed, only the IAM policy needs to be added for it to work (see https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html)
    - Windows images already have SSM pre-installed, only the IAM policy needs to be added for it to work (see https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html)


Related issue https://github.com/weaveworks/eksctl/issues/2743



### Checklist
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
